frontend/src/components/projects/ProjectsDetailedContent.js

```diff
@@ -46,6 +46,11 @@ const ProjectsDetailedContent = ({projects, projectsByID, isFetching, get}) => {
                 <Descriptions.Item label="Targeted End Date" span={2}>{project.targeted_end_date}</Descriptions.Item>
                 <Descriptions.Item label="Comments" span={4}>{project.comments}</Descriptions.Item>
             </Descriptions>
+            <Descriptions bordered={true} size="small" title="Associated Samples" style={{marginTop: "24px"}}>
+                {project.samples && project.samples.map(sample => {
+                  return (<div> <Link to={`/samples/${sample.id}`}> • {sample.name} </Link> </div>);
```

Comment:
We'll need to have something a bit more elaborated than that. Maybe this is just a stub, but i will comment to make sure we do not forget. We will need to include in this section a table with the sample name and sample kind minimally with a way to order and filter them by name and kind. The business plan lists cohort and volume, but i am not sure if this was updated to the latest agreement with the stakeholders... I would add those 2 fields too (cohort and volume) since there may be projects where cohorts are important to organize the samples within and volume maybe be a relevant information for PMs.